src/xrSound/SourceCache.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+#include "xrCommon/xr_unordered_map.h"
+#include "xrCommon/xr_deque.h"
+#include "SoundRender.h"
```

Comment:
Usually, we separate different modules with a blank line.
Here, xrCommon and xrSound folders are considered as different modules.
```suggestion
#include "xrCommon/xr_deque.h"

#include "SoundRender.h"
```